components/operator-service/src/main/java/com/wso2telco/dep/operatorservice/service/BlackListWhiteListService.java

```diff
@@ -56,68 +59,62 @@ public BlackListWhiteListService()
 	 * sign if not all the process fails. The entire process run as a single
 	 * transaction .
 	 */
-	public void blacklist(BlackListDTO dto) throws BusinessException {
+	public APIBlacklistWhitelistResponseDTO blacklist(BlacklistWhitelistDTO dto) throws BusinessException {
 
-		String[] msisdns = dto.getUserMSISDN();
-		MSISDNValidationDTO msisdnValidationDTO = new MSISDNValidationDTO();
-
-		final String apiID_ = dto.getApiID();
-		final String apiName_ = dto.getApiName();
-		final String userId_ = dto.getUserID();
-
-
-			try{
-
-				msisdnValidationDTO.setValid(Arrays.asList(msisdns));
-				msisdnValidationDTO.setValidationRegex(dto.getValidationRegex());
-				msisdnValidationDTO.setValidationPrefixGroup(dto.getValidationPrefixGroup());
-				msisdnValidationDTO.setValidationDigitsGroup(dto.getValidationDigitsGroup());
-				msisdnValidationDTO.process();
+		List<String> valid = new ArrayList<>();
+		List<String> invalid = new ArrayList<>();
+		APIBlacklistWhitelistResponseDTO responseDTO = new APIBlacklistWhitelistResponseDTO(0,0);
```

Comment:
Either set the values through constructor or use setters (not both at the same time if there is a valid reason) in APIBlacklistWhitelistResponseDTO. No point of setting 0s in the constructor and use setters later to set values.